src/Microsoft.Diagnostics.Runtime/datatarget.cs

```diff
@@ -2045,16 +2058,40 @@ public unsafe bool GetThreadContext(uint threadID, uint contextFlags, uint conte
     internal unsafe class LiveDataReader : IDataReader
     {
         #region Variables
+        private int _originalPid;
+        private IntPtr _snapshotHandle;
+        private IntPtr _vaCloneHandle;
```

Comment:
Can this be renamed "_cloneHandle" for consistency?  ClrMD doesn't really use Hungarian notation for variable names.